apps/expo/src/app/user/account-settings.tsx

```diff
@@ -76,90 +23,86 @@ const styles = StyleSheet.create({
     flex: 4,
     margin: 4,
     alignSelf: 'center',
-    color: colors.grey
+    color: colors.silver
   },
   navigationListItemChevron: {
     alignSelf: 'center',
-    color: colors.grey
-  },
-  nudgeBtn: {
-    margin: 12
+    color: colors.silver
   }
-})
+});
 
-interface NavigationListItem {
-  title: string,
-  iconName: string,
-  onPress: () => void;
-}
+const tailwindClasses = {
+  mainTile: 'm-4 p-4 rounded-lg bg-bb-dark-gray',
+  mainTileTitle: 'text-2xl text-slate-200',
+  mainTileItem: 'flex-row',
+  navigationListItemLabel: 'flex-auto m-2 self-center text-slate-200',
+  navigationListItemChevron: 'self-center text-slate-200'
+};
+
+const accountItems: NavigationListItem[] = [
+  { title: 'Personal Data', iconName: 'user', onPress: () => router.push('/user/personal-data') },
+  { title: 'Achievements', iconName: 'award', onPress: () => router.push('/user/achievements') },
+  { title: 'Activity History', iconName: 'history', onPress: () => router.push('/user/activity-history') },
+  { title: 'Workout Progress', iconName: 'chart-pie', onPress: () => router.push('/user/workout-progress') }
+];
+
+const otherItems: NavigationListItem[] = [
+  { title: 'Contact Us', iconName: 'envelope', onPress: () => router.push('/contact-us') },
+  { title: 'Privacy Policy', iconName: 'shield-alt', onPress: () => router.push('/privacy-policy') }
+];
 
 const AccountSettings = () => {
   const [notificationsSwitchEnabled, setIsEnabled] = useState(false);
   const toggleNotificationSwitch = () => setIsEnabled(previousState => !previousState);
-
   const { data } = api.user.byId.useQuery({ id: 1 });
   const user = data; // alias for readability
 
-  const accountItems: NavigationListItem[] = [
-    { title: 'Personal Data', iconName: 'user', onPress: () => router.push('/user/personal-data') },
-    { title: 'Achievements', iconName: 'award', onPress: () => router.push('/user/achievements') },
-    { title: 'Activity History', iconName: 'history', onPress: () => router.push('/user/activity-history') },
-    { title: 'Workout Progress', iconName: 'chart-pie', onPress: () => router.push('/user/workout-progress') }
-  ];
-
-  const otherItems: NavigationListItem[] = [
-    { title: 'Contact Us', iconName: 'envelope', onPress: () => router.push('/contact-us') },
-    { title: 'Privacy Policy', iconName: 'shield-alt', onPress: () => router.push('/privacy-policy') }
-  ]
-
   return (
-    <SafeAreaView style={[styles.screenContainer, { flex: 1 }]}>
-      <View style={styles.userTile}>
-        <View style={ { flexDirection: 'row', alignItems: 'center' }}>
-          <FontAwesome5 name="user" size={48} style={{padding: 22, color: colors.purple}}/>
-        </View>
-        <View style={[styles.userTileUpper, { flex: 3 }]}>
-          <Text style={styles.userFullName}>{user?.name}</Text>
-          <Text style={styles.userProgram}>Program</Text>
+    <SafeAreaView className='flex-1 bg-bb-slate-100' style={{backgroundColor: '#1e1e1e', flex: 1}}>
```

Comment:
QUESTION: Why do we still use the style attribute if we already have the corresponding class names applied?

Reply:
I believe he has some issue with the nativewind styling does not apply ?  means that using className and style w it won't works, while style does

Reply:
I had similar issues and what fixed it was cleaning my entire build/prebuild and rebuilding again (this was on iOS).
The commands I used were `pnpm expo prebuild --clean` and `pnpm expo build:ios --no-build-cache` (for android it will be `pnpm expo build:android --no-build-cache`. Both these commands were ran in the root of the expo folder.

Reply:
Yeah the tailwind classes aren't working for some of the elements. I'll try rebuilding like you said.

Reply:
Yea we can leave it as is for nowl